content/en/docs/faq/applications/remove-built-in-apps.md

```diff
@@ -0,0 +1,39 @@
+---
+title: "Remove the Built-in Applications in KubeSphere"
```

Comment:
Remove the Built-in Applications
->
Remove Built-in Apps

Note that, in most cases, we need to make sure the value of `title` and `linkTitle` are the same.

Reply:
Noted.